java/arcs/sdk/Particle.kt

```diff
@@ -11,14 +11,16 @@
 
 package arcs.sdk
 
+import arcs.core.host.api.Particle
+
 /**
  * Interface used by [ArcHost]s to interact dynamically with code-generated [Handle] fields
  * used by [Particle]s.
  */
-typealias HandleHolder = arcs.core.host.HandleHolder
+typealias HandleHolder = arcs.core.host.api.HandleHolder
 
 /** Base interface for all particles. */
-typealias Particle = arcs.core.host.Particle
+typealias Particle = Particle
```

Comment:
Is this needed/meaningful?

Reply:
Not sure how that got there, but I reverted it